assets/js/utils/get-query.js

```diff
@@ -1,29 +1,38 @@
-export default function getQuery( attributes ) {
+export default function getQuery( attributes, name ) {
 	const { categories, columns, orderby, rows } = attributes;
 
 	const query = {
 		status: 'publish',
 		per_page: rows * columns,
 	};
 
-	if ( categories ) {
+	if ( categories && categories.length ) {
 		query.category = categories.join( ',' );
 	}
 
-	if ( 'price_desc' === orderby ) {
-		query.orderby = 'price';
-		query.order = 'desc';
-	} else if ( 'price_asc' === orderby ) {
-		query.orderby = 'price';
-		query.order = 'asc';
-	} else if ( 'title' === orderby ) {
-		query.orderby = 'title';
-		query.order = 'asc';
-	} else if ( 'menu_order' === orderby ) {
-		query.orderby = 'menu_order';
-		query.order = 'asc';
-	} else {
-		query.orderby = orderby;
+	if ( orderby ) {
+		if ( 'price_desc' === orderby ) {
```

Comment:
Long term would it be better to pass in both `order` and `orderby` or is there an advantage to passing it as a single param?

Reply:
We only have one ordering input (where it offers "price, low to high", "price, high to low", etc), that's why we need to translate that into the two order/orderby params.

Reply:
Got it 👍